compiler/desugared/from_surface.ml

```diff
@@ -1142,6 +1142,21 @@ let process_scope_use_item
   | Surface.Ast.Rule rule -> process_rule precond scope ctxt prgm rule
   | Surface.Ast.Definition def -> process_def precond scope ctxt prgm def
   | Surface.Ast.Assertion ass -> process_assert precond scope ctxt prgm ass
+  | Surface.Ast.DateRounding (r, _) ->
+    let scope_uid = scope in
+    let scope : Ast.scope = ScopeName.Map.find scope_uid prgm.program_scopes in
+    let r =
+      match r with
+      | Surface.Ast.Increasing -> Ast.Increasing
+      | Surface.Ast.Decreasing -> Ast.Decreasing
+    in
+    let new_scope =
+      { scope with scope_options = Ast.DateRounding r :: scope.scope_options }
```

Comment:
What happens with several (possibly conflicting) rounding modes in the same scope ? We may want it to be an error.